packages/docs/src/routes/docs/concepts/resumable/index.mdx

```diff
@@ -57,15 +62,18 @@ Qwik solves the above issue by serializing the event listeners into DOM like so:
 
 Qwik still needs to collect the listener information, but this step is done as part of the SSR/SSG. The results of SSR/SSG are then serialized into HTML so that the browser does not need to do anything to resume the execution. Notice that the `on:click` attribute contains all of the information to resume the application without doing anything eagerly.
 
-1. Qwikloader sets up a single global listener instead of many individual listeners per DOM element. This step can be done with no application code present.
+1. [Qwikloader](https://qwik.builder.io/docs/advanced/qwikloader/) sets up a single global listener instead of many individual listeners per DOM element. This step can be done with no application code present.
 2. The HTML contains a URL to the chunk and symbol name. The attribute tells Qwikloader which code chunk to download and which symbol to retrieve from the chunk.
+[//]: # (by looking at the compiled code, symbol likely to be a function, is "retrieve" the proper term ?)
 3. Finally, to make all of the above possible, Qwik's event processing implementation understands asynchronicity which allows insertion of asynchronous lazy loading.
+[//]: # (I can’t really figure the meaning of this and what is synchronous lazy-loading. What does it mean for Qwik to "understand asynchronicity" ?)
```

Comment:
Yes, all lazy loading is async, so this doesn't make sense, I agree

I would change it to "Finally, to make all of the above possible, Qwik's event processing implementation understands asynchronicity which allows it to lazy load closuers, automatically.